src/main/java/woowacourse/crewniverse/milkyway/repository/CrewRepository.java

```diff
@@ -0,0 +1,13 @@
+package woowacourse.crewniverse.milkyway.repository;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import woowacourse.crewniverse.milkyway.domain.Crew;
+
+public interface CrewRepository extends JpaRepository<Crew, Long> {
+
+    @Query("SELECT c FROM Crew c LEFT JOIN Attendance a ON c = a.crew AND a.date = :date WHERE a.id IS NULL")
+    List<Crew> findByAbsentedCrew(LocalDate date);
```

Comment:
- `a.id IS NULL`이 왜 필요한가요 ?
- 아래와 같은 메서드명은 어떨까요?
```suggestion
    List<Crew> findAbsentCrewsByDate(LocalDate date);
```

Reply:
> a.id IS NULL이 왜 필요한가요 ?

출석폼을 제출하지 않은 크루를 확인해야 하는데, attendance에 저장되지 않은 크루들을 조회하기 위해서 필요합니다 

> 아래와 같은 메서드명은 어떨까요?

👍

Reply:
위 답변과 이어지는데, `null`을 사용하는 것이 껄끄러워서 위 답변에서 설명한 대로 진행하면 어떨까요?

Reply:
온갖 시도를 다 해봤는데, 현재 Attendance 테이블은 출석을 한 사람만 데이터가 저장되고 있기 때문에 출석을 하지 않는 크루의 정보는 가지고 있지 않아서 Crew 테이블을 주측으로 left join해야 해당 정보를 가지고 올 수 있네요 😢

Reply:
오 그러네요, Crew를 기반으로 없는 것을 찾으려면 null을 사용해야겠습니다, 설명 고마워요 🤩